spec/requests/sessions_spec.rb

```diff
@@ -3,25 +3,29 @@
 RSpec.describe '/session controllers' do
   describe "#create, delete" do
     context 'post request sign in' do
```

Comment:
`context` or `describe` should describe only one action so each `create` and `delete` should have separate context